Hotfix v2.5.1 (#1940)
precice/precice: CHANGELOG.md

All notable changes to this project will be documented in this file. For future plans, see our [Roadmap](https://www.precice.org/fundamentals-roadmap.html).

## 2.5.1

- Changed error to warning when trying to add a zero-value column to `V` matrix in QN-acceleration (Backport of https://github.com/precice/precice/pull/1863)
- Fixed compatibility with libxml version 2.12.0. (https://github.com/precice/precice/pull/1886)

> - Added a warning for empty IQN matrix, but keep the simulation going. This allows to start from a zero initial state that only later changes (e.g., an opening valve in a flow simulation). (https://github.com/precice/precice/pull/1895)

Reviewer: The entry sounds a bit, as if the warning is new, but actually we changed the behavior from error -> warning. Not sure if this is completely sure for someone not knowing the old behavior.

Reviewer 2: Technically, the warning is new. From the user perspective, this can indeed be difficult to understand. How about:

```suggestion
- Changed an error to warning in case of an empty IQN matrix, keeping the simulation going. This allows to start from a zero initial state that only later changes (e.g., an opening valve in a flow simulation). (https://github.com/precice/precice/pull/1895)
```

Author: Good point. I changed it and pushed the new version.
